src/status_im/common/data_store/wallet.cljs

```diff
@@ -48,6 +51,23 @@
       (update :testPreferredChainIds chain-ids-set->string)
       (dissoc :watch-only?)))
 
+(defn- rpc->balances-per-chain
+  [token]
+  (-> token
+      (update :balances-per-chain
+              update-vals
+              #(-> %
+                   (update :raw-balance money/bignumber)
+                   (update :balance money/bignumber)))
+      (update :balances-per-chain update-keys (comp utils.number/parse-int name))))
+
+(defn rpc->tokens
+  [tokens]
+  (-> tokens
+      (update-keys name)
+      (update-vals #(cske/transform-keys csk/->kebab-case %))
+      (update-vals #(mapv rpc->balances-per-chain %))))
```

Comment:
> About updating the `:balance` key, I remeber the desktop team said we have that key because of retrocompatibility, but shouldn't be used. So I'd say better not using that key unless we need it

Noted. Thanks @ulisesmac. I have removed the transform (to BigNumber) for the `balance` key. 👍